README.md

```diff
@@ -56,15 +52,14 @@ into which the project was cloned and run `stack build`.
 The [`stack` system](https://docs.haskellstack.org/en/stable/README/) is
 required before `cardano-sl` can be built.
 
-Running the nodes
------------------
-
-After the project is built, run it:
-
-- open *new terminal window*, run `tmux` in it (`sudo apt-get install tmux` or else according to your distro)
-- open *new terminal window*, run `export WALLET_TEST=1; ./scripts/launch.sh` in it
```

Comment:
@ChShersh because everything you mentioned is in the documentation.  I think the `README`, having largely been superseded by the docs page, should mention things we don't there, of which there are not many.